main.go

```diff
@@ -165,6 +184,7 @@ func start(cfg config.Configuration, port int, kubeConfig *rest.Config, devmode
 	// client here thus making the mapper not reach out to the target cluster at all.
 	mapper := meta.NewDefaultRESTMapper([]schema.GroupVersion{})
 	mapper.Add(authz.SchemeGroupVersion.WithKind("SelfSubjectAccessReview"), meta.RESTScopeRoot)
+	//	mapper.Add(auth.SchemeGroupVersion.WithKind("TokenReview"), meta.RESTScopeRoot)
```

Comment:
why is this commented out ?

Reply:
Basically, it didn't help to make TokenReview. In theory, it should.